scheduler/crates/api/src/event/subscribers/mod.rs

```diff
@@ -53,7 +53,6 @@ impl Subscriber<CreateEventUseCase> for CreateSyncedEventsOnEventCreated {
             Ok(synced_calendars) => synced_calendars,
             Err(e) => {
                 error!("Unable to query synced calendars from repo: {:?}", e);
-                println!("Unable to query synced calendars from repo: {:?}", e);
```

Comment:
`error!()` also prints a log, so no need to do it twice

I think before, `error!` wasn't working properly in localhost so `println!` was needed